bindings/py/cpp_src/CMakeLists.txt

```diff
@@ -132,11 +132,11 @@ target_link_libraries(${algorithms_shared_lib} PRIVATE
 target_compile_options(${algorithms_shared_lib} PUBLIC ${INTERNAL_CXX_FLAGS})
 target_compile_definitions(${algorithms_shared_lib} PRIVATE ${COMMON_COMPILER_DEFINITIONS})
 target_include_directories(${algorithms_shared_lib} PRIVATE 
-        ${PYTHON_INCLUDE_DIRS}
-        ${pybind11_INCLUDE_DIRS}
-        ${CORE_LIB_INCLUDES} 
-        ${EXTERNAL_INCLUDES}
+        ${CORE_LIB_INCLUDES}
         ${PROJECT_SOURCE_DIR}
+	SYSTEM ${PYTHON_INCLUDE_DIRS}
```

Comment:
Thanks @dkeeney for the advice!, 

> We could catch all of them in one place by including SYSTEM at the beginning of the list at the point where the EXTERNAL_INCLUDES list is built which is line 93 of extern/bootstrap.cmake.  

I tried this, by adding `"SYSTEM"` to the `set(EXTERNAL_INCLUDES ...) ` but that didn't work as intended. 

> On line 450 and 464 of src/CMakeLists.txt, I did not specify SYSTEM for EXTERNAL_INCLUDES. 

..so I fixed all ocurences of EXTERNAL_INCLUDES in `target_include_dirs()` with SYSTEM, that works. 
But not for `httplib`, unfortunately. Is it included in the EXTERNAL_INCLUDES, or does it use a different means of including?